pepper-apis/src/main/resources/db-changes/schema/DDP-5096-study-rgp-export-last-completed.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                   http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+
+    <changeSet author="charlotte" id="20210315-study-rgp-export-last-completed">
+        <createTable tableName="rgp_export">
```

Comment:
Did you speak with other folks (e.g. Zim) about this approach? Did they give their blessing on having study-specific code/tables?

Reply:
I would prefer we not name tables with study identifiers.  Can we changes this to `export_operation` or some such, and remove `rgp` prefixes in general?

Reply:
That makes sense.  Plus this also makes it easier to make this available to other studies if at some point we decide to do that.